lib/rom/sql/types/pg.rb

```diff
@@ -5,21 +5,49 @@ module ROM
   module SQL
     module Types
       module PG
-        Sequel.extension(:pg_json)
+        # UUID
+
+        UUID = Types::Strict::String.constrained(format: Regexp.new('^([0-9a-f]{8})-([0-9a-f]{4})-([0-9a-f]{4})-([0-9a-f]{2})([0-9a-f]{2})-([0-9a-f]{12})$'))
```

Comment:
parens can be omitted, also it's common to use `\A` and `\z` in ruby instead of `^` and `$` because of some security issues, see [this SO post](http://stackoverflow.com/questions/577653/difference-between-a-z-and-in-ruby-regular-expressions) for details